### ISSUE TITLE
Wrong install size calculated for updates, blocking them

### ISSUE THREAD

**Issue by melvyn2** [derrod/legendary#318]
## Platform

Operating system and version: Arch Linux
Legendary version (`legendary -V`): `legendary version "0.20.6", codename "A Red Letter Day"`

## Expected Behavior
During an update, the update size should be calculated from the total size increase, not total game size.

## Current Behavior
The total game size is used and blocks installation with not enough disk space.
```
 legendary update 9d2d0eb64d5c44529cece33fe2a46482
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Preparing download...
[Core] INFO: Parsing game manifest...
[Core] INFO: Install path: /run/media/melvyn/Windoz/Program Files/Epic Games/GTAV
[cli] INFO: Install size: 103908.68 MiB
[cli] INFO: Download size: 4216.50 MiB (Compression savings: 0.9%)
[cli] INFO: Reusable size: 546.00 MiB (chunks) / 99111.28 MiB (unchanged / skipped)
[cli] INFO: Installation requirements check returned the following results:
[cli] WARNING: This game is not marked for offline use (may still work).
[cli] CRITICAL: Not enough available disk space! 65530.95 MiB < 107705.29 MiB
[cli] ERROR: Installation cannot proceed, exiting.
```

## Steps to Reproduce
1. Have a game bigger than remaining disk space
2. Attempt an update

**Comment by nutterthanos** [comment]
Uh this was already fixed in master and previously reported with #317

**Comment by derrod** [comment]
Duplicate of #317
